### PR TITLE
Add customization guide for next.js app router

### PR DESCRIPTION
**Pull request by mateuszo**
This is an app router version of the existing pages router guide.

### CODE REVIEW

File: docs/content/guides/4.customization/1.index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Alokai Customization Guide
+layout: default
+---
+
+# Alokai Customization Guide
+
+On this page you'll find a set of guides that will help you to understand the advanced concepts of Alokai and how to customize the application to your requirements.
```
Comment (mattmaribojoc) [inline docs/content/guides/4.customization/1.index.md:8]:
```suggestion
On this page, you'll find a set of guides that will help you understand some advanced concepts of Alokai and how to customize the application to your requirements. 
```

File: docs/content/guides/4.customization/1.pages-router/1.index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+title: Alokai Customization Guide
+layout: default
+---
+
+# Alokai Customization Guide
+
+Alokai is not a cookie-cutter solution, it is meant to be able to handle even the most complex use cases. This guide will take you through the most common customization scenarios. We aim to cover end-to-end implementation of realistic business requirements.
```
Comment (mattmaribojoc) [inline docs/content/guides/4.customization/1.pages-router/1.index.md:8]:
```suggestion
Alokai is not a cookie-cutter solution, it is meant to flexible enough to handle even the complex use cases. This guide will take you through the most common customization scenarios. We aim to cover end-to-end implementation of realistic business requirements.
```

File: docs/content/guides/4.customization/2.app-router/2.ui-customizations.md [hunk 3]
```diff
@@ -0,0 +1,380 @@
+---
+title: Basic UI customizations
+layout: default
+navigation:
+  icon: tabler:number-1-small
+---
+
+# UI customizations
+
+Out of the box, an Alokai application ships with default design system built on top of [Storefront UI](https://docs.storefrontui.io). While this helps ensure that your application looks good instantly, it's likely one of the first things you'll want to customize.
```
Comment (mattmaribojoc) [inline docs/content/guides/4.customization/2.app-router/2.ui-customizations.md:10]:
```suggestion
Out of the box, an Alokai application ships with a default design system built on top of [Storefront UI](https://docs.storefrontui.io). While this helps ensure that your application looks good instantly, it's likely one of the first things you'll want to customize. 
```

File: docs/content/guides/4.customization/2.app-router/2.ui-customizations.md [hunk 4]
```diff
@@ -0,0 +1,380 @@
...
+The logo should look better now. However, buttons in the navbar have become invisible. That's because they are white.
+
+::tip 
+#title
+An Extra Challenge
+#default
+Within the `NavbarTop` component, find which tailwind class is responsible for making the buttons white and remove it.
+::
+
+## Adjusting theme colors
```
Comment (mattmaribojoc) [inline docs/content/guides/4.customization/2.app-router/2.ui-customizations.md:72]: Probably also a good time to link to https://docs.storefrontui.io/v2/customization/theming

File: docs/content/guides/4.customization/2.app-router/2.ui-customizations.md [hunk 5]
```diff
@@ -0,0 +1,380 @@
...
+
+![PLP customizations](./images/plp-customization.webp)
+
+To check your solution, you can look at [our implementation](https://github.com/vsf-customer/extensibility-demo-v2/blob/main/apps/storefront-unified-nextjs/components/product-card-vertical.tsx).
+
+## Facet search on PLP
+
+Now let's try something more ambitious - we'll extend facet's (aka filters) behavior. We want to be able to filter/search through the facets.
+
+Before jumping to the solution I encourage you to think about how would you do this yourself. Investigate the application and which parts you need to modify.
```
Comment (mattmaribojoc) [inline docs/content/guides/4.customization/2.app-router/2.ui-customizations.md:221]:
```suggestion
Before jumping to the solution, think about how would you do this yourself. Investigate the application and which parts you need to modify.
```

**Review by mattmaribojoc — APPROVED** [review]
Looks great! Few small grammar changes, but great work

File: docs/content/guides/4.customization/2.app-router/2.ui-customizations.md (hunk 4 above)
Comment (mateuszo) [reply docs/content/guides/4.customization/2.app-router/2.ui-customizations.md:72]: It's linked down below ;)

**Pull request merged**